.github/workflows/rich-code-nav.yml

```diff
@@ -0,0 +1,44 @@
+name: Rich Code Navigation
+
+on:
+  push:
+  pull_request:
+    branches:
+      - main
+      - stable
+
+env:
+  dotnet_sdk_version: '5.0.100'
+  DOTNET_SKIP_FIRST_TIME_EXPERIENCE: true
+
+jobs:
+  build:
+    runs-on: windows-latest
```

Comment:
Does it not work on Linux? :trollface: 

BTW I no longer use *-latest, I prefer to pin (e.g. windows-2019)

Reply:
A good question I have no answer to. Haven't found any information on it, and docs only for MS employees.